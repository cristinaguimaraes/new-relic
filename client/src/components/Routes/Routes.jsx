import React from 'react';
import { Route, Switch } from 'react-router-dom';

import { Home } from '../../pages/Home';
import { AppsByHost } from '../../pages/AppsByHost';


const Routes = () => {
  
  return (
    <div >
      <Switch>
        <Route exact path='/' component={Home} />
        {/*<Route path='/TopApps' component={TopApps}/>*/}
        <Route path='/AppsByHost' component={AppsByHost}/>
      </Switch>
    </div>
  )
};


export { Routes };
