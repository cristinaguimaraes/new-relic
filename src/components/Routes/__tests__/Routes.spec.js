import { Routes } from "../Routes";
import { shallow } from "enzyme";
import React from "react";

describe("Routes component", () => {
  test("should render properly", () => {
    expect(
      shallow(
        <Routes onAddApp={jest.fn()} onShowToastMessage={jest.fn()}>
          Test
        </Routes>
      )
    ).toMatchSnapshot();
  });
});
