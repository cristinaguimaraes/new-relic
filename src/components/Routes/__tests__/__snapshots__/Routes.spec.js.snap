// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes component should render properly 1`] = `
<BrowserRouter>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      path="/apps-by-host"
    />
    <Route
      component={[Function]}
      path="/top-apps-by-host/:id"
    />
  </Switch>
</BrowserRouter>
`;
